Sprintfinity3902/Interfaces/IRoomLoader.cs

```diff
@@ -2,5 +2,10 @@
 {
     public interface IRoomLoader
     {
+        public void Initialize(IRoom room);
```

Comment:
Should these be declared as "public"? Or should they just be void like other interfaces?

Reply:
Interfaces should not include access modifiers. Thank you for pointing this out. I will change it!